attribute_set/models/attribute_attribute.py

```diff
@@ -247,13 +246,12 @@ def domain_change(self):
             except ValueError:
                 raise ValidationError(
                     _(
-                        """ "{}" is an unvalid Domain name.\n
+                        """ "%(domain)s" is an unvalid Domain name.\n
```

Comment:
@JasminSForgeFlow Don't get back to old-style python formatting.

You should let the former notation. But you should call format on the translation result (and not inside translation function)

Reply:
Updated, Thanks